CHANGELOG.rst

```diff
@@ -25,6 +25,12 @@ Fixed
 
   Reported by John Arnold.
 
+* Fix a bug where sensor watch queues were not deleted after sensor container process was shut
```

Comment:
Should we add a small bash script which deletes those and tell users in upgrade notes on how to run it?